org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/ResourceUtils.java

```diff
@@ -116,4 +126,41 @@ public static String fixURI(URI uri) {
 		String uriString = uri.toString();
 		return uriString.replaceFirst("file:/([^/])", "file:///$1");
 	}
+
+	/**
+	 * @param project
+	 * @param className
+	 * @return
+	 * @throws JavaModelException
+	 * @throws UnsupportedEncodingException
+	 */
+	public static String getURI(IProject project, String className) throws JavaModelException, UnsupportedEncodingException {
```

Comment:
This method belongs to the test plugin for now. 

Code makes many assumptions that are acceptable in the context of a test case, but would require more robust checks otherwise.